package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "squishjs",
-  "version": "0.4.9",
+  "version": "0.5",
   "description": "squish & unsquish stuff",
   "scripts": {
-    "test": "node testRunner.js && node ./test/test.js"
+    "test": "node test.js"
```

Comment:
I think the change here is a mistake to what was in master
We should use the testRunner here and the ./test/test.js file should probably be changed to use the testRunner instead